cmake/IgnCodeCheck.cmake

```diff
@@ -32,9 +32,15 @@ function(ign_setup_target_for_codecheck)
 
   message(STATUS "Adding codecheck target")
 
+  # Each include dir neeads an -I flag
```

Comment:
```suggestion
  # Each include dir needs an -I flag
```

Reply:
Done in https://github.com/ignitionrobotics/ign-cmake/pull/186/commits/64deb31c6ad5f602cff25a87e8783229b4d735d2.